spec/features/staff_can_create_a_communal_defect_spec.rb

```diff
@@ -65,6 +67,10 @@
       expect(page).to have_content('Communal Area')
       expect(page).to have_content('33-50 Hackney Street, communal entrance')
     end
+
+    within('.evidence') do
+      expect(page).to have_content('Some uploaded evidence')
```

Comment:
Should we check if the evidence has been uploaded too somehow?

Reply:
Great shout, pushing an improved test now.